pandas/io/html.py

```diff
@@ -637,41 +688,30 @@ def _build_doc(self):
                 raise XMLSyntaxError("no text parsed from document", 0, 0, 0)
         return r
 
-    def _parse_tbody(self, table):
-        return table.xpath('.//tbody')
+    def _parse_thead_tr(self, table):
+        rows = []
 
-    def _parse_thead(self, table):
-        return table.xpath('.//thead')
+        for thead in table.xpath('.//thead'):
+            rows.extend(thead.xpath('./tr'))
```

Comment:
Does this work if the `thead` has multiple rows?

Reply:
yes. (Previous code returned a list of <thead>, treating it as a list of lists. New code effectively flattens the return value.)